composer.json

```diff
@@ -27,6 +27,11 @@
             "Tightenco\\Collect\\": "src/Collect"
         }
     },
+    "autoload-dev": {
```

Comment:
Is this necessary? Tests seemed to have worked without it, so I'm curious about the value.

Reply:
In this case, it's not necessary because the classes in `src` and `tests` are the same namespace.
If the test classes namespace is different from source classes, this ```autoload-dev``` option in `composer.json` is necessary.